index.html

```diff
@@ -466,8 +466,126 @@ <h4>Amanda Jepson</h4>
           </div>
 
         </div>
+         <div class="section-title" data-aos="zoom-out">
+        <p>Executive Team Members</p>
+      </div>
+     <div class="row">
+
+          <div class="col-lg-3 col-md-6 d-flex align-items-stretch">
+            <div class="member" data-aos="fade-up">
+              <div class="member-img">
+                <img src="assets/img/team/team-1.jpg" class="img-fluid" alt="">
+                <div class="social">
+                  <a href=""><i class="icofont-twitter"></i></a>
+                  <a href=""><i class="icofont-facebook"></i></a>
+                  <a href=""><i class="icofont-instagram"></i></a>
+                  <a href=""><i class="icofont-linkedin"></i></a>
+                </div>
+              </div>
+              <div class="member-info">
+                <h4>Walter White</h4>
```

Comment:
Team sections changes are incorrect, please change them to the club's current team then make your PR.